common/test/acceptance/tests/studio/test_studio_tabs.py

```diff
@@ -99,11 +99,87 @@ def test_user_can_reorder_static_tabs(self):
         self.pages_page.set_field_val("Display Name", "First")
         self.pages_page.save()
         self.pages_page.add_static_page()
-        self.pages_page.drag_and_drop_first_static_page_to_last()
+        self.pages_page.drag_and_drop()
         self.pages_page.refresh_and_wait_for_load()
         static_tab_titles = self.pages_page.static_tab_titles
         self.assertEqual(
             static_tab_titles,
             ['Empty', 'First'],
             'Order should be:["Empty", "First] but getting {} from the page'.format(static_tab_titles)
         )
+
+    def test_user_can_reorder_builtin_tabs(self):
+        """
+        Scenario: Users can reorder built-in pages
+            Given I have opened the pages page in a new course
+                Then the built-in pages are in the default order
+            When I drag the first page to the last
+                Then the built-in pages are switched
+            And I reload the page
+                Then the built-in pages are switched
+        """
+        default_order = ['Home', 'Course', 'Discussion', 'Wiki', 'Progress']
+        new_order = ['Home', 'Course', 'Wiki', 'Progress', 'Discussion']
+        default_tabs_order = self.pages_page.default_tab_titles
+        self.assertEqual(
+            default_tabs_order,
+            default_order,
+            'Tabs are not in the default order'
+        )
+        self.pages_page.drag_and_drop(default_tab=True)
+        default_tabs_order = self.pages_page.default_tab_titles
+        self.assertEqual(
```

Comment:
Just a nit
we are using default_tabs_order = self.pages_page.default_tab_titles twice in the test.

Since there is no reuse of variable so we can just remove these lines and use the actual method name in assertion like 
self.assertEqual(
            self.pages_page.default_tab_titles,
            default_order,
            'Tabs are not in the default order'
        )